File: old/src/scanners/CitraFlatpakSource.js
const CitraSource = require("./CitraSource.js");
const CitraFlatpakGame = require("../games/CitraFlatpakGame.js");

const USER_DIR = process.env["HOME"];

class CitraFlatpakSource extends CitraSource{

	static name = "Citra (Flatpak)";
	static gameClass = CitraFlatpakGame;

	configPath = `${USER_DIR}/.var/app/org.citra_emu.citra/config/citra-emu/qt-config.ini`;

}

module.exports = CitraFlatpakSource;

File: gali/sources/citra/citra_flatpak_source.py
from gali.sources.citra.citra_native_source import CitraNativeSource
from gali.sources.citra.citra_flatpak_game import CitraFlatpakGame
from gali.utils.locations import HOME

class CitraFlatpakSource(CitraNativeSource):

    name: str = "Citra (Flatpak)"
    game_class: type[CitraFlatpakGame] = CitraFlatpakGame
    config_path: str = f"{HOME}/.var/app/org.citra_emu.citra/config/citra-emu/qt-config.ini"
